modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/api-header.mustache

```diff
@@ -22,11 +22,24 @@ namespace {{this}} {
 
 using namespace {{modelNamespace}};
 
-class {{declspec}} {{classname}}
+class {{declspec}} I{{classname}}
```

Comment:
I think this part should be encapsulated with the new mustache variable, so that current users are not impacted when